flocker/acceptance/test_api.py

```diff
@@ -404,6 +442,24 @@ def restarted(record):
         creating_dataset.addCallback(created_dataset)
         return creating_dataset
 
+    def test_current(self):
+        """
+        The current container endpoint includes a currently running container.
+        """
+        creating = self._create_container()
+
+        def created(result):
+            cluster, data = result
+            data[u"running"] = True
+
+            def in_current():
+                current = cluster.current_containers()
+                current.addCallback(lambda result: data in result[1])
+                return current
+            return loop_until(in_current)
```

Comment:
:question: There's a `Cluster.wait_for_dataset` method. Perhaps it'd be useful to have a `wait_for_container` method.